odoo_sh/getting_started/settings.rst

```diff
@@ -0,0 +1,194 @@
+:banner: banners/odoo-sh.jpg
+
+==================================
+Settings
+==================================
+
+Overview
+========
+
+The settings allow you to manage the configuration of your project.
+
+.. image:: ./media/interface-settings.png
+   :align: center
+
+Project name
+============
+
+The name of your project.
+
+.. image:: ./media/interface-settings-projectname.png
+   :align: center
+
+This is the domain that will be used to access your production database.
+
+The domain to access your staging and development builds will contain this name as well.
```

Comment:
Addresses of your staging and development builds are derived from this name and assigned automatically.